script.js

```diff
@@ -0,0 +1,263 @@
+        this._removeMain();
+        this.init('englishShift');
+        this.properties.language = "russian";
+        this.elements.textArea.value = this.properties.value;
+        }
+        else if (this.properties.language == "russian" && this.properties.shift){
+        this._removeMain();
+        this.init('russianShift');
+        this.properties.language = "russian";
+        this.elements.textArea.value = this.properties.value;
+        }
+    },
+
+    addEventListnerKeybord() {
+        document.addEventListener('keydown', (event) => this._keyDownHendler(event));
+        document.addEventListener('keyup', (event) => this._keyUpHendler(event));
+    },
+
+    _keyDownHendler(event) {
+        event.preventDefault();
+        let eventclick = new Event ("click");
+        let eventmousedown = new Event ("mousedown");
+        const keybordCharCods = [192, 49, 50, 51, 52, 53, 54, 55, 56, 57, 48, 189, 187,
+        8, 9, 81, 87, 69, 82, 84, 89, 85, 73, 79, 80, 219, 221, 220, 46, 20, 65, 83, 68, 70,
+        71, 72, 74, 75, 76, 186, 222, 13, 16, 90, 88, 67, 86, 66, 78, 77, 188, 190, 191, 16, 38, 17, 0, 18, 32, 0, 0, 0, 37, 40, 39];
+        let arrKeysKeyboard = [...document.querySelectorAll('.keyboard__key')];
+        let indexKeyPresed = keybordCharCods.indexOf(event.keyCode);
+        if (event.keyCode == 20) {arrKeysKeyboard[indexKeyPresed].classList.toggle('keyboard__key--active');
+        arrKeysKeyboard[indexKeyPresed].dispatchEvent(eventclick);}
+        else if (indexKeyPresed == -1) {}
+        else if (event.keyCode == 16) { 
+            if (event.repeat) {} else {
+            arrKeysKeyboard[indexKeyPresed].dispatchEvent(eventmousedown);
+            arrKeysKeyboard[indexKeyPresed].classList.add('keyboard__key--active');}
+        }
+        else if (event.keyCode == 17) { this.properties.ctrl = true;
+            arrKeysKeyboard[indexKeyPresed].classList.add('keyboard__key--active')
+        }
+        else if (event.keyCode == 18) { arrKeysKeyboard[indexKeyPresed].classList.add('keyboard__key--active');
+        (this.properties.ctrl == true) ? this.toggleLanguage(): this.properties.ctrl = false;}
+        else {
+        arrKeysKeyboard[indexKeyPresed].dispatchEvent(eventclick);
+        arrKeysKeyboard[indexKeyPresed].classList.add('keyboard__key--active');
+        }
+    },
+
+    _keyUpHendler(event) {
+        event.preventDefault();
+        let eventmouseup = new Event ("mouseup");
+        const keybordCharCods = [192, 49, 50, 51, 52, 53, 54, 55, 56, 57, 48, 189, 187,
+        8, 9, 81, 87, 69, 82, 84, 89, 85, 73, 79, 80, 219, 221, 220, 46, 20, 65, 83, 68, 70,
+        71, 72, 74, 75, 76, 186, 222, 13, 16, 90, 88, 67, 86, 66, 78, 77, 188, 190, 191, 16, 38, 17, 0, 18, 32, 0, 0, 0, 37, 40, 39];
+        let arrKeysKeyboard = [...document.querySelectorAll('.keyboard__key')];
+        let indexKeyPresed = keybordCharCods.indexOf(event.keyCode);
```

Comment:
use destructuring for event

Reply:
Sory, may be you can show example, or give materials